tools/generate-countries.ipynb

```diff
@@ -915,6 +962,163 @@
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## -- Create list of Hungary data sets"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def create_html_for_Hungary(regions, wwwroot, expiry_hours=10):\n",
+    "    \"\"\"If a data set has been created within expire_hours from now, do not update it, but return \n",
+    "    immediately.\"\"\"\n",
+    "    does_wwwroot_exist(wwwroot)\n",
+    "        \n",
+    "    start_time = time.time()\n",
+    "    skipped = 0\n",
+    "    \n",
+    "    \n",
+    "    for i, region in enumerate(regions):\n",
+    "        name = f'Hungary-{region}'\n",
+    "        metadata = MetadataRegion(name)\n",
+    "        if metadata.last_updated_hours_ago() < expiry_hours:\n",
+    "            print(f\"Skipping Hungary - was updated {metadata.last_updated_hours_ago():.1f} hours ago\")\n",
+    "            skipped += 1\n",
+    "            continue\n",
+    "        \n",
+    "        # metadata to be used when we create html pages \n",
+    "        metadata['source'] = \"https://github.com/sanbrock/covid19/blob/master/datafile.csv\"\n",
+    "        metadata['category'] = \"Hungary\"\n",
+    "        cases, _, region_label = get_country_data(\"Hungary\", region=region)\n",
+    "        \n",
+    "\n",
+    "        metadata['max-deaths'] = 0\n",
+    "        metadata['max-cases'] = int(cases[-1])\n",
+    "        metadata['region'] = region\n",
+    "        \n",
+    "        one_line_summary = f\"Hungary: {region}\"\n",
+    "        metadata['one-line-summary'] = one_line_summary  # used as title in table\n",
+    "        # compute number of infections in last week\n",
+    "        metadata['cases-last-week'] = int(get_cases_last_week(cases))\n",
+    "    \n",
+    "\n",
+    "        try:    \n",
+    "            print(f\"Processing {i+1}/{len(regions)} [{time.time()-start_time:4.0f}s]\")\n",
+    "            ipynb_name = create_ipynb_for_hungary(region=region, templatename=\"template-hungary.ipynb\", wwwroot=wwwroot)\n",
+    "            html_name = nbconvert_ipynb2html(ipynb_name, wwwroot=wwwroot)\n",
+    "            metadata['html-file'] = html_name\n",
+    "            metadata['ipynb-name'] = ipynb_name\n",
+    "            print(f\"Mark {region} as updated\")\n",
+    "            metadata.mark_as_updated()\n",
+    "\n",
+    "        except Exception as e:\n",
+    "            print(f\"Error for {region}\", end='')\n",
+    "            print(e)\n",
+    "            raise\n",
+    "            \n",
+    "        sys.stdout.flush()\n",
+    "\n",
+    "    print(f\"Created {len(regions)-skipped} (skipped {skipped}) notebooks and html versions in \" + \\\n",
+    "          f\"{time.time()-start_time} seconds\")\n",
+    "    sys.stdout.flush()\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "metadata": {},
+   "outputs": [
+    {
```

Comment:
You should clear the notebook before committing it. In Jupyter, Cell -> All Outputs -> Clear, or Kernel -> Restart & Clear Output